frontend/src/components/core/Block/Block.tsx

```diff
@@ -185,7 +186,7 @@ class Block extends PureComponent<Props> {
     )
 
     const elementType = element.get("type")
-    const isEmpty = elementType === "empty"
+    const isEmpty = elementType === "empty" || elementType === "favicon"
```

Comment:
I think the variable name should be changed also. `isEmptyOrFavicon`?

Reply:
Renamed this (and the html class name) to `isHidden`